homeassistant/components/kostal_plenticore/config_flow.py

```diff
@@ -21,6 +21,7 @@
     {
         vol.Required(CONF_HOST): str,
         vol.Required(CONF_PASSWORD): str,
+        vol.Optional("Service Code"): str,
```

Comment:
Consider adding a constant for the service code (used multiple times)

Reply:
Where is the best place to add it? In const.py of kostal_plenticore?